src/client/clientmap.cpp

```diff
@@ -978,9 +978,10 @@ void ClientMap::renderMapShadows(video::IVideoDriver *driver,
 	std::vector<DrawDescriptor> draw_order;
 
 
-	int count = 0;
-	int low_bound = is_transparent_pass ? 0 : m_drawlist_shadow.size() / total_frames * frame;
-	int high_bound = is_transparent_pass ? m_drawlist_shadow.size() : m_drawlist_shadow.size() / total_frames * (frame + 1);
+	std::size_t count = 0;
+	std::size_t meshes_per_frame = m_drawlist_shadow.size() / total_frames + 1;
```

Comment:
is this meant to be rounded?

Reply:
This is meant to be rounded up, so the algorithm is greedy, taking more meshes in the first frames and leaving fewer meshes to the last frame.

Reply:
This formula is slightly skewed towards the first frames but is the cheapest and simplest to do the job.